.github/workflows/tests_gpu_python.yml

```diff
@@ -150,13 +150,21 @@ jobs:
         run: |
           python -m pip install cutensornet-cu${{ matrix.cuda_version }}
 
-      - name: Checkout PennyLane for release build
-        if: inputs.pennylane-version == 'release'
+      - name: Checkout PennyLane for release or latest build
+        if: inputs.pennylane-version == 'release' || inputs.pennylane-version == 'latest'
         uses: actions/checkout@v4
         with:
           path: pennylane
           repository: PennyLaneAI/pennylane
 
+      - name: Switch to latest build of PennyLane
+        if: inputs.pennylane-version == 'latest'
+        run: |
+          cd pennylane
+          git fetch --all
```

Comment:
Does it work with `git fetch origin master`?

Reply:
Wondering the same thing.